std/utf.d

```diff
@@ -4,7 +4,7 @@
     Encode and decode UTF-8, UTF-16 and UTF-32 strings.
 
     UTF character support is restricted to
-    $(D '\u0000' &lt;= character &lt;= '\U0010FFFF').
+    `'\u0000' &lt;= character &lt;= '\U0010FFFF'`.
```

Comment:
This will have different output than the before, since backtick will html encode its content in current D (both implementation and spec) and `$(D ...)` does not.

So the former will actually output `<= character` and the new will show, to the end user, as `&lt;= character`. You can't use html entities or tags inside the backticks.